builder_test.go

```diff
@@ -145,13 +136,58 @@ func Test_Builder(t *testing.T) {
 				Spacing(""),
 			".match{$i}{$j}\n1 2{{\\{first\\}}}\n2 0{{second {$i}}}\n3 0{{{|\\\\a\\||}}}\n* *{{{1}}}",
 		},
+		{
+			"attributes",
+			NewBuilder().
+				Text("Attributes for variable expression ").
+				Expr(
+					Var("i").
```

Comment:
Why not variadic attributes for method Var()?

Reply:
1. Only one variable is possible
2. Not this PR scope